docs/releases/1.10.txt

```diff
@@ -328,6 +328,11 @@ Miscellaneous
 
 * The ``makemigrations --exit`` option is deprecated in favor of the
   :djadminopt:`--check` option.
+* In Django 2.0 default error views will raise a
+  ``TemplateNotFoundError`` if a non existing custom template name
+  is passed as argument. A ``PendingDeprecationWarning`` is emitted
```

Comment:
I would avoid referring to the actual warning class since this comment will require adjustment in Django 1.11 where this will actually be a `DeprecationWarning`.